Source/WebCore/platform/mediastream/libwebrtc/LibWebRTCProviderCocoa.cpp

```diff
@@ -102,10 +102,11 @@ void LibWebRTCProviderCocoa::setActive(bool value)
 
 bool WebRTCProvider::webRTCAvailable()
 {
-#if (PLATFORM(IOS) || PLATFORM(VISION)) && !PLATFORM(IOS_SIMULATOR)
+#if PLATFORM(IOS) || PLATFORM(VISION)
     return true;
 #else
-    return !!webrtc::setApplicationStatus;
+    ASSERT(!!webrtc::setApplicationStatus);
```

Comment:
I think that you want the opposite - assert in embedded case, and `return !!webrtc::setApplicationStatus;` on macOS.

Not sure what's up with watchOS and tvOS here.

Reply:
Watch & TV don't have libwebrtc at the moment I believe.

Reply:
I suggest:
`ASSERT_WITH_MESSAGE(!!webrtc::setApplicationStatus, "Failed to find or load libwebrtc");`